sound/soc/intel/common/soc-acpi-intel-tgl-match.c

```diff
@@ -221,6 +245,25 @@ static const struct snd_soc_acpi_link_adr tgl_3_in_1_mono_amp[] = {
 	{}
 };
 
+static const struct snd_soc_acpi_link_adr tgl_olympic_mlk[] = {
```

Comment:
please remove references to 'olympic mlk', we don't use such code names (including in the commit message)

We can call this tgl_sdw_rt711_link1_rt1308_link2_rt1715_link0.